treebrowser/README

```diff
@@ -33,7 +33,7 @@ Features
 
 Requirements
 ============
-* GTK >= 2.8.0
+* GTK >= 3.10.0
```

Comment:
You apparently don't actually need 3.10 (only 3.0 for CSS stuff).
And if you did, you'd need to change the version check in the build system